vllm/engine/llm_engine.py

```diff
@@ -1250,12 +1314,19 @@ def step(self) -> List[Union[RequestOutput, EmbeddingRequestOutput]]:
             if self.scheduler_config.is_multi_step:
                 self.cached_scheduler_outputs[0] = SchedulerOutputState()
 
+            current_step: Optional[int] = self._current_step(
+                seq_group_metadata_list)
+            # minus 1 because we updated the step count above
```

Comment:
Ditto. Where the step count is updated?

Reply:
in `finish_step()` above. I have updated the comment so it is clear.